lib/addons/cloudwatch-adot-addon/index.ts

```diff
@@ -42,10 +47,11 @@ export const enum cloudWatchDeploymentMode {
 /**
  * Defaults options for the add-on
  */
-const defaultProps = {
+const defaultProps: CloudWatchAdotAddOnProps = {
     deploymentMode: cloudWatchDeploymentMode.DEPLOYMENT,
     namespace: 'default',
-    name: 'adot-collector-cloudwatch'
+    name: 'adot-collector-cloudwatch',
+    metricsNameSelectors: ['apiserver_request_.*', 'container_memory_.*', 'container_threads', 'otelcol_process_.*']
```

Comment:
default selection seems to make sense, but I don't see anything other otelcol_process* metrics in cloudwatch. Expected to see pod and container memory for each container in all namespaces.